examples/nxterm/nxterm_internal.h

```diff
@@ -92,8 +92,8 @@
 #    define CONFIG_EXAMPLES_NXTERM_BPP 8
 #  elif !defined(CONFIG_NX_DISABLE_16BPP)
 #    define CONFIG_EXAMPLES_NXTERM_BPP 16
-//#elif !defined(CONFIG_NX_DISABLE_24BPP)
-//#    define CONFIG_NXTERM_BPP 24
+#elif !defined(CONFIG_NX_DISABLE_24BPP)
```

Comment:
```suggestion
#  elif !defined(CONFIG_NX_DISABLE_24BPP)
```
I didn't make this change because was not sure why the `CONFIG_NXTERM_BPP​ ​24` case was comment out.

Reply:
Done.